test/support/runner.rb

```diff
@@ -0,0 +1,48 @@
+require 'pathname'
+
+module RegexpParserTest
+  class Runner
+    def initialize(arguments, warning_whitelist)
+      @arguments = arguments
+      @warning_whitelist = warning_whitelist
+    end
+
+    def run
+      test_status = nil
+
+      warnings =
```

Comment:
It doesn't look like the assignment to the `warnings` lvar isn't actually needed.